agilent_infiniium_scope.py

```diff
@@ -227,28 +227,30 @@ def numavg(self, c, navg = None):
 
     @setting(131, channel = 'i', level = 'v', returns = 'v{level}')
     def trigger_at(self, c, channel, level=None):
-        """Get or set the trigger source and the trigger voltage for edge mode
-         triggering.
-        Channel must be one of 0 (AUX), 1, 2, 3, 4, 5(LINE).
+        """Get or set the trigger source and voltage for edge triggering.
+
+        Channel must be one of 0 (AUX), 1, 2, 3, 4, or 5 (LINE).
+        If trigger source is 5 (LINE) the level will be ignored.
         """
         dev = self.selectedDevice(c)
         if channel not in [0, 1, 2, 3, 4, 5]:
-            raise Exception('Select valid trigger channel')
+            raise ValueError('Invalid trigger channel: {}'
+                             'Valid channels are [0, 1, 2, 3, 4, 5]'
+                             '0 for AUX, 5 for LINE'.format(channel))
```

Comment:
When you have neighboring string literals like this, they get concatenated without additional spaces or newlines, so you'll want to add some punctuation and spaces:
```python
'Invalid trigger channel: {}. '
'Valid channels are [0, 1, 2, 3, 4, 5] '
'(0 for AUX, 5 for LINE).'.format(channel)
```